wp-includes/js/offline-page-handling.template.js

```diff
@@ -0,0 +1,26 @@
+/* global OFFLINE_PAGE_URL */
+( () => {
```

Comment:
My old JS habits are carrying through here. We only need to wrap in a `{` block since the `const` variables are scoped lexically. No need for IIFE.